docs/source/common/production_inference.rst

```diff
@@ -46,3 +64,32 @@ that returns a scripted module which you can save or directly use.
     torch.jit.save(script, "model.pt")
 
 It is recommended that you install the latest supported version of PyTorch to use this feature without limitations.
+
+Once you have the exported model, you can run it in Pytorch or C++ runtime:
+
+.. code-block:: python
+
+    inp = torch.rand(1, 64)
+    scripted_module = torch.jit.load("model.pt")
+    output = scripted_module(dummy_input)
+
+---------------
+
+********************
+Using Python Runtime
+********************
+
+You can also load the saved checkpoint inside a Python runtime and use it as a regular :class:`torch.nn.Module`.
+
+.. code-block:: python
+
+    model = SimpleModel()
+
+    # train it
+    trainer = Trainer(gpus=2)
+    trainer.fit(model, train_dataloader, val_dataloader)
+
+    # use model after training or load weights and drop into the production system
+    model.eval()
+    with torch.no_grad():
+        y_hat = model(x)
```

Comment:
This came up previously on the "Lightning for production" discussion here: https://www.reddit.com/r/MachineLearning/comments/qgqq07/d_is_pytorch_lightning_production_ready/

One of the risks that using the whole LightningModule as an nn.Module introduces is all of the dependencies it brings in, which may not be suitable for a production environment.

I think the simplest option for users is to extract their nn.Modules & weights from their LightningModule to be used for inference in production.

Reply:
made some updates! mind review?